scaling_performance/optimizing_storage.adoc

```diff
@@ -135,11 +137,6 @@ In a scaled/HA {product-title} registry cluster deployment:
 * All NAS storage (excluding {gluster-native}/{gluster-external} GlusterFS as it uses an object storage interface) are
 not recommended for {product-title} Registry cluster deployment with production workloads.
 
-[WARNING]
-====
-include::install_config/topics/NFS_note.adoc[]
-====
-
 [[metrics]]
 ==== Metrics
 
```

Comment:
Overall this document needs to be looked at more closely. 
Under Logging and Metrics (subsections) we have "warning" notes: "Corruption may occur when using NFS to back hosted logging with production workloads."
- We should remove these (and align the warning with other messaging (in other places) and to the table above.
    - The table above says we support RWO, and RWX (as configurable options for both logging and metrics). 
    - In locations (other docs) we call out NFS backed file systems should not be used for various reasons (we should link to these notes and callouts). 

Under the Logging subsection, we have the note: 
- "NFS does not guarantee read-after-write consistency and is not recommended for applications which require it." 
    - This should be removed, as not every application requires this (so it may not be relevant). 
- "Applications that depend on writing to the same, shared NFS export may experience issues with production workloads."
    - We should update this to state: 
        - Applications are responsible for knowing and understanding the storage requirements for their application. It is on the application/development/deployment team to understand how an application work with the backed/provided storage, to ensure that issues do not occur with an application scales or interacts with the storage layer.